remo/base/views.py

```diff
@@ -42,8 +43,16 @@ def login_failure(self, msg=''):
 def main(request):
     """Main page of the website."""
     featured_rep = utils.latest_object_or_none(FeaturedRep)
-    feed = feedparser.parse(settings.PLANET_URL)
-    planet_entries = feed.entries[:3]
+
+    res = False
+    try:
+        res = requests.get(settings.PLANET_URL, timeout=settings.PLANET_MAX_TIMEOUT).text
+    except requests.exceptions.ConnectionError:
+        planet_entries = ''
+    if res:
```

Comment:
You can remove this check by assigning an empty string in the response in the `except` clause.